docs/api/requests/definePlayerHandler.md

```diff
@@ -28,21 +28,21 @@ For IMDb series (provided by Cinemeta), the video ID is formed by joining the Me
 
 ``action`` - set in the `extra` object; a string defining the user action, can be either: `start`, `end`, `pause` or `resume`.
 
-``duration`` - set in the `extra` object; int specifying the full duration of the video in **milliseconds**.
+``duration`` - set in the `extra` object; string specifying the full duration of the video in **milliseconds**.
 
-``currentTime`` - set in the `extra` object; int in *milliseconds* specifying the progress from the start of the video when the user took the action. 
+``currentTime`` - set in the `extra` object; string in *milliseconds* specifying the progress from the start of the video when the user took the action. 
```

Comment:
I'm not sure about the "string" change. It is a regular number, string might give the impression that it has to be parsed from a String, e.g. `"123"`

Reply:
cuz it has to be parsed from a string. in the SDK anything extracted from the request URL is a string. 
for example if u request
`/player/{type}/{id}/action=play&currentTime=100&duration=100000.json`
u will get 
```
extraArgs = {
action = "play",
currentTime = "100",
duration = "100000"
}
```
so if u wanna use them as int u will have to convert them into integers first

Reply:
Ok, I'm resolving this question then.